microcosm_flask/conventions/health.py

```diff
@@ -60,49 +63,63 @@ class Health:
     def __init__(self, graph, include_build_info=True):
         self.graph = graph
         self.name = graph.metadata.name
+        self.optional_checks = dict()
+        self.checks = dict()
+
         if include_build_info:
-            self.checks = dict(
+            self.checks.update(dict(
```

Comment:
I decided to keep these as default checks for the moment to avoid possible disruption. These are used pretty frequently, and they're relatively cheap to begin with.

We'll always be able to remove later.